apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/index.tsx

```diff
@@ -15,8 +15,64 @@ import FeatureFlagsSettings from "./FeatureFlagsSettings";
 import EnableLearnPageStagingUrlToggle from "./EnableLearnPageStagingUrlToggle";
 import OnboardingAppInstallDebugButton from "./OnboardingAppInstallDebug";
 import EnableStagingNftMetadataServiceToggle from "./EnableStagingNftMetadataServiceToggle";
+import { addTokens, getCryptoCurrencyById } from "@ledgerhq/coin-framework/currencies";
+import { Button } from "@ledgerhq/react-ui";
+
+function convertERC20([
+  parentCurrencyId,
+  token,
+  ticker,
+  magnitude,
+  name,
+  ledgerSignature,
+  contractAddress,
+  disableCountervalue,
+  delisted,
+  countervalueTicker,
+]: any): any {
```

Comment:
Any way to avoid `any` here ?

Reply:
It's just a temporary code, it will help QA to test functionnality so it will be delete after test